docs/tools_cheatsheet/Makefile.md

```diff
@@ -0,0 +1,142 @@
+**A simple Makefile**
+
+```Makefile
+    CC  =	gcc
+
+    NAME  =	binary_name
+
+    CFLAGS  =	 -Wpedantic -pedantic-errors -w  -Wextra  -Wall -Iinclude

```

Comment:
You don't need this for a basic makefile
So, you can remove it or simply add one flag and explain it as well.